programs/zstdcli.c

```diff
@@ -838,6 +847,7 @@ int main(int const argCount, const char* argv[])
                 if (longCommandWArg(&argument, "--target-compressed-block-size=")) { targetCBlockSize = readSizeTFromChar(&argument); continue; }
                 if (longCommandWArg(&argument, "--size-hint=")) { srcSizeHint = readSizeTFromChar(&argument); continue; }
                 if (longCommandWArg(&argument, "--output-dir-flat=")) { outDirName = argument; continue; }
+                if (longCommandWArg(&argument, "--output-dir-mirrored=")) { outMirroredDirName = argument; continue; }
```

Comment:
Is there a reason why this command is slightly different ?
`--output-dir-mirror` vs `--output-dir-mirrored` ?

Reply:
good catch! Thanks!